Fix command to create AKS cluster

c-w opened the pull request:
Running the `az aks create` command listed in the tutorial fails on the latest azure-cli (version 2.0.31) with the following error:

```
Deployment failed. Correlation ID: {some-uuid}. PutControlPlane error
```

This is likely related to https://github.com/Azure/AKS/issues/38

Specifying a particular kubernetes version fixes the issue and the cluster is created as expected.

azurecla commented:
Hi __@Saisang__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-zhecai). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

c-w committed:
Fix command to create AKS cluster

Running the `az aks create` command listed in the tutorial fails on the latest
azure-cli (version 2.0.31) with the following error:

```
Deployment failed. Correlation ID: {some-uuid}. PutControlPlane error
```

This is likely related to https://github.com/Azure/AKS/issues/38

Specifying a particular kubernetes version fixes the issue and the cluster is
created as expected.
